src/main/migrations/changesets/top_level_output.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="chrisl" id="top_level_output">
+        <comment>Adds a TOP_LEVEL_OUTPUT field to SYMBOL so that the DB can track which symbols represent top-level outputs.</comment>
```

Comment:
:+1:  to @Horneth - why can't a sub-workflow's outputs simply be treated as a standard call's outputs?